CHANGELOG.md

```diff
@@ -111,6 +111,9 @@ to avoid letting the user edit the immutable one.
 * `jj rebase -r` now accepts `--insert-after` and `--insert-before` options to
   customize the location of the rebased revisions.
 
+* A new revset `reahable(srcs, domain)` will return all commits that are
+  reachable from `srcs` within `domain`.
+
```

Comment:
Just noticed there's a typo here, and also this is in the wrong section (should be under the unreleased changes).

Reply:
This got fixed earlier by #3730, thanks Benjamin!